recipes/dcmtk/all/patches/0001-cmake-use-conan-packages.patch

```diff
@@ -71,7 +82,7 @@
        message(STATUS "Info: DCMTK ICONV support will be enabled")
        set(WITH_LIBICONV 1)
      else() # turn off library if library path not set
-@@ -170,7 +170,7 @@
+@@ -170,7 +170,7 @@ if(WIN32 AND NOT MINGW)
```

Comment:
This addition seems out of place 🤔

Reply:
My git diff gave me in another format than existing one, so that's why the change.
I refactored the patch to not create additional chnages.